examples/xml-operations/xml_operations.description

```diff
@@ -1,11 +1,10 @@
-// `+` does concatenation.
-// `==` does deep equals.
-// `foreach` iterates over each item.
-// `x[i]` gives i-th item (empty sequence if none).
-// `x.id` accesses required attribute named `id`:
-// result is error if there is no such attribute
-// or if x is not a singleton.
-// `x?.id` accesses optional attribute named `id`:
-// result is () if there is no such attribute.
-// Langlib lang.xml provides other operations.
-// Mutate an element using `e.setChildren(x)`.
+// <ul>
+// <li>`+` does concatenation.</li>
+// <li>`==` does deep equals.</li>
+// <li>`foreach` iterates over each item.</li>
+// <li>`x[i]` gives i-th item (empty sequence if none).</li>
+// <li>`x.id` accesses required attribute named `id`: result is error if there is no such attribute or if x is not a singleton.</li>
+// <li>`x?.id` accesses optional attribute named `id`: result is () if there is no such attribute.</li>
+// <li>Langlib lang.xml provides other operations.</li>
+// <li>Mutate an element using `e.setChildren(x)`.</li>
+// </ul>
```

Comment:
Isn't it odd for the entire description to be a list?

Reply:
Lets's revert this https://github.com/ballerina-platform/ballerina-distribution/pull/2568